masterlist.yaml

```diff
@@ -11678,6 +11678,21 @@ plugins:
       - crc: 0x87290B79
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
 
+  - name: 'Vigil Enforcer Distribution.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/31991/'
+        name: 'The Vigilants'' New Clothes - A Creation Club Integration Mod'
+    tag:
+      - Actors.CombatStyle
```

Comment:
The NPC combat style change doesn't appear to be particularly important to the mod since the mod changes it to a ranged style when the NPC appears to only use melee weapons:

![image](https://user-images.githubusercontent.com/69141501/154434406-2e020488-4c59-445a-85ef-264e0127d6c1.png)

```suggestion
```